.github/workflows/ppa-automation.yaml

```diff
@@ -3,6 +3,8 @@ on:
   push:
     branches:
       - 'releases/**'
+    tags:
+      - 'v[0-9]+.[0-9a-z]+.[0-9a-z]+'
```

Comment:
why do we need the letters in the `[]` blocks?

Reply:
I didn't want to exclude the possibility of release numbering weirdness, like `2.6.1a` or something, but we've never actually tagged such a release.